RTradeLtd/ipfs-orchestrator
# TEM-62 daemon: implement mvp functionality

## Testing Instructions

1. `make testenv`
2. add an entry to the `hosted_ip_fs_private_networks` with a name, eg `test-network`
3. `make config`, edit config settings as desired (most importantly the database part)
4. `./ipfs-orchestrator -dev daemon`
5. start up a network: `./ipfs-orchestrator -dev ctl StartNetwork Network=test-network`
6. see stats: `./ipfs-orchestrator -dev ctl NetworkStats Network=test-network`
7. stop the network: `./ipfs-orchestrator -dev ctl StopNetwork Network=test-network`
